yarn-project/ethereum/src/test/start_anvil.ts

```diff
@@ -22,6 +22,7 @@ export async function startAnvil(l1BlockTime?: number): Promise<{ anvil: Anvil;
         anvilBinary,
         port: ethereumHostPort,
         blockTime: l1BlockTime,
+        codeSizeLimit: 0x7000,
```

Comment:
Rollup.sol now very large and overflows contract size limits :( will try and bring it down in future commits